docs/source/sycamore/SDK_APIs.rst

```diff
@@ -1,10 +1,10 @@
-Aryn Partitioning Service APIs
+Aryn SDK APIs
```

Comment:
We shouldn't have any Aryn SDK stuff in the Sycamore docs now - should we remove this?

Reply:
Removed.